Tutorial.md

```diff
@@ -11,477 +10,247 @@ Gist is a paste. A snippet of text. Not unrelated to [GitHub Gists][].  To set s
 > + [API Blueprint Map][map]
 > + [Tools for API Blueprint][Tooling Section]
 
-## Gist Fox API
-Without any further ado our *Gist Fox API* blueprint starts like this:
+## API Blueprint
 
-    FORMAT: 1A
+The start of the blueprint looks as follows:
 
-    # Gist Fox API
-    Gist Fox API is a **pastes service** similar to [GitHub's Gist](http://gist.github.com).
+```markdown
+FORMAT: 1A
 
-What have we just done? Let's break it line by line:
+# Polls
 
-#### Metadata
+Polls is a simple API allowing consumers to view polls, and vote in them.
+```
 
-    FORMAT: 1A
+## Metadata
 
-Depending on the tools you will be using with your blueprint you might want to start your blueprint file with a metadata explicitly stating the API Blueprint version you want to use like in this case the version 1A.
+The blueprint starts wih a metadata section. In this case we have specified that `FORMAT` has the value of `1A`. The format keyword is used to specify the version of the API Blueprint. In this case 1A.
```

Comment:
Remove "that."

> ...we have specified `FORMAT` has...

Also, specify is used twice. You may make it something like:

> The format keyword denotes the version of the API Blueprint, which in this case is `1A`.

Be sure to make `1A` inline code like your first example of it.